scripts/guests.js

```diff
@@ -10,4 +10,21 @@
                 Add park area id as state
 
         Return HTML String
-*/
\ No newline at end of file
+*/
+
+import { getGuestsList } from "./database.js"
```

Comment:
I believe that Jayme used a different name for this function. Please update with the appropriate function name